src/app/my25/[userId]/page.jsx

```diff
@@ -6,6 +6,16 @@ import { AlertBox } from '@/components/AlertBox';
 import { FavoriteMovieList } from './components/FavoriteMovieList';
 import { ChooseMovieModal } from './components/ChooseMovieModal';
 
+function labelFromListLength(length) {
+    if (length > 25) {
```

Comment:
If we do make the change to my earlier comment, this would need to be updated to use the the constants.

Reply:
Done!